src/containers/Patient.js

```diff
@@ -11,41 +12,50 @@ import huddleGroupProps from '../prop-types/huddle_group';
 import huddleProps from '../prop-types/huddle';
 import riskServiceProps from '../prop-types/risk_service';
 import riskAssessmentProps from '../prop-types/risk_assessment';
+import riskAssessmentBreakdownProps from '../prop-types/risk_assessment_breakdown';
 
 import { fetchPatient } from '../actions/patient';
 import { fetchHuddles, selectHuddle, addPatientToHuddle } from '../actions/huddle';
 import { fetchRiskServices, selectRiskService } from '../actions/risk_service';
-import { fetchRiskAssessments } from '../actions/risk_assessment';
+import { fetchRiskAssessments, fetchRiskAssessmentBreakdown, selectRiskAssessment } from '../actions/risk_assessment';
 
 export class Patient extends Component {
   constructor(...args) {
     super(...args);
 
-    this.state = { loading: false };
+    this.state = {
+      loading: false
+    };
   }
 
   componentWillMount() {
-    if (this.props.riskServices.length === 0) { this.props.fetchRiskServices(); }
-
     if (this.props.selectedPatient == null ||
         this.props.params.patient_id !== this.props.selectedPatient.id) {
       // --patient not yet fetched--
       this.setState({ loading: true });
-      // fetch patient
-      this.props.fetchPatient(this.props.params.patient_id).then(() => {
-        this.props.fetchRiskAssessments(this.props.params.patient_id, this.props.selectedRiskService.id);
+      // fetch patient and patient data
+      let patientPromise = this.props.fetchPatient(this.props.params.patient_id);
+      let riskServicePromise = this.props.fetchRiskServices();
+
+      Promise.all([patientPromise, riskServicePromise]).then(() => {
```

Comment:
You can also use `axis.all` here the exact same way. (I'm pretty sure it's just a wrapper) 

Not a blocker just letting you know it's a thing.